lib/config/rules/react.js

```diff
@@ -128,8 +128,14 @@ module.exports = {
   'react/jsx-max-depth': 'off',
   // Limit maximum of props on a single line in JSX
   'react/jsx-max-props-per-line': 'off',
-  // Prevent usage of .bind() and arrow functions in JSX props
-  'react/jsx-no-bind': 'error',
+  // Prevent usage of .bind() in JSX props
+  'react/jsx-no-bind': [
+    'error',
+    {
+      allowArrowFunctions: true,
+      allowFunctions: true,
```

Comment:
What does this mean is not allowed?

Reply:
only `bind` calls are not allowed. The other config options have to do with ignoring refs/dom elements.

Reply:
And they default to false:

```
  "ignoreDOMComponents": <boolean> || false,
  "ignoreRefs": <boolean> || false,
  "allowArrowFunctions": <boolean> || false,
  "allowFunctions": <boolean> || false,
  "allowBind": <boolean> || false
```

Reply:
I think we may as well allow bind calls then, there's no real difference of `this.handleFoo.bind(this)` and `() => this.handleFoo()`